src/neptune/metadata_containers/project.py

```diff
@@ -286,6 +289,10 @@ def fetch_runs_table(
         verify_type("sort_by", sort_by, str)
         verify_type("ascending", ascending, bool)
 
+        if states is not None:
```

Comment:
From what I see, `states` will never be `None`, right? The type hint for the `as_list` function doesn't seem to be adequate, or am I missing something?
https://github.com/neptune-ai/neptune-client/blob/22a0fc72d9d6983a42a05a9dd04ceb79b53a2f1a/src/neptune/internal/utils/__init__.py#L193

Reply:
I know that the `as_list` function is not part of your PR, but just noticed that its confusing return type hint requires additional, unnecessary `if` statement

Reply:
Thanks a lot! Fixed I think